src/features/creator/creator.view.tsx

```diff
@@ -44,6 +44,19 @@ const CreatorView: React.FC = () => {
     creatorStoreActions.change(value);
   };
 
+  const getEditorState = (): string => {
+    return `https://4markdown.com`;
+  };
+
+  const handleLinkClick = (): void => {
+    const editorStateLink = getEditorState();
```

Comment:
Question, why it's changed?

Reply:
These changes have been applied because the previous version of the button opened a new tab in the browser, instead of a new window, as indicated by the description in the icon title.